kinetic/distribution.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  fedora:
+  - '22'
+  - '23'
```

Comment:
We can simply not support distributions which are not at least available as alpha when we start the release process. Therefore only 23 and 24 seems feasible to me. Also adding them later is not an option (the same with newer Ubuntu distros.)